docs/source/intro.rst

```diff
@@ -11,4 +11,57 @@ As a programmatic interface to Ansible, the SDK lets you:
 * Marshall parameters for operations to different formats.
 * Pass data in native Python structures to and from Ansible.
 
-You can execute Ansible content with the SDK in any combination of local or remote subprocesses with or without containerization.
\ No newline at end of file
+SDK architecture
+----------------
+
+Ansible SDK runs in your Python application and takes the following inputs:
+
+Project
+    Logical collection of Ansible playbooks, credentials, and inventories.
+Credentials
+    Authentication with host systems.
+Inventory
+    List of host systems on which automation jobs run.
+
+With those inputs, Ansible SDK makes multi-threaded, asynchronous calls to perform automation runs against your project payload.
```

Comment:
Is it actually multithreaded?

Reply:
@Akasurde Do you have any insights on this one? I was thinking it might be an idea to look at some of the tests to see if I could observe it in action but maybe it's just better to let you respond. Thx.

Reply:
Probably just asynchronous. There's no such thing as “muti-threaded calls”.

Reply:
For the record though I did get "multi-threaded" from some internal collateral. I didn't just make that up. :smile_cat: